README.md

```diff
@@ -1,17 +1,53 @@
 # cilium-sysdump
 
-# Overview
- The cilium-sysdump tool collects all information in a cluster required to
- troubleshoot issues with Cilium and Hubble.
+The cilium-sysdump tool collects all information in a cluster required to troubleshoot issues with Cilium and Hubble.
 
-# Prerequisites
-- Requires Python >= 2.7.*
-- Requires `kubectl`. 
-- `kubectl` should be pointing to your cluster before running the tool.
+## Prerequisites
+- Python >= 2.7
+- kubectl
+- pip (required only if you intend to rebuild the zip file)
 
-# (Optional) Re-build zip file
-    make build
+## Using cilium-sysdump
 
-# Run:
-    python cilium-sysdump.zip
+Optionally rebuild the zip file with:
 
+```sh
+$ pip install pycodestyle pyflakes
+$ make build
+```
```

Comment:
Please, remove these lines, this is only useful for development or when creating a new release. Users are only expected to download a [release](https://github.com/cilium/cilium-sysdump/releases) and run `python cilium-sysdump.zip`.